contracts/vending/VendingMachine.sol

```diff
@@ -80,20 +90,20 @@ contract VendingMachine is Ownable, IReceiveApproval {
             _wrappedTokenSupply;
```

Comment:
What do you think about adding few additional checks?
* that there is no remainder of division when `ratio` is calculated (by calculating `_tTokenAllocation` back for example)
* check contracts and supplies `_tToken.totalSupply() >= _tTokenAllocation` and `_wrappedToken.totalSupply() >= _wrappedTokenSupply`

Reply:
Added the supplies check. The remainder check for ratio calculation doesn't add much, and it would simply move the remainder problem somewhere else (specifically, to the address that funds the vending machine with T)